packages/svelte/src/reactivity/set.js

```diff
@@ -117,7 +117,11 @@ export class ReactiveSet extends Set {
 	}
 
 	clear() {
-		if (super.size !== 0) {
+		var size = super.size;
```

Comment:
why create this variable? we're only using `size` in one place

Reply:
Because clearing it will result in size being 0.